bloom/block_amd64.s

```diff
@@ -2,6 +2,15 @@
 
 #include "textflag.h"
 
+#define salt0 0x47b6137b
```

Comment:
I'm curious where these salt values come from.

Reply:
They come from the the parquet specs https://github.com/apache/parquet-format/blob/master/BloomFilter.md, I don't know how they generated them in the first place.